src/guiguts/application.py

```diff
@@ -836,6 +836,17 @@ def init_html_menu(self) -> None:
         html_menu.add_button("~Unmatched HTML Tags...", unmatched_html_markup)
         html_menu.add_button("HTML5 ~Validator (online)...", html_validator_check)
```

Comment:
Remove this line? I think the HTML validator was the first one you added, and hadn't standardized yet on the (www) suffix?

<img width="241" alt="image" src="https://github.com/user-attachments/assets/cfe9b140-4b4c-466d-869e-ff8fb949cab2" />

Otherwise, they both worked for me!

I didn't know whether the open file would automatically be uploaded when the option was chosen, or whether it would require the user to navigate to the file. Since it's the latter (i.e. further input is required, unlike with the help options), perhaps the `...` is appropriate to indicate that?

Reply:
The first one is the one that submits the HTML file for validation via the API (built-in validation for GG). The second one just opens the W3C page in your browser, like the CSS validator link.

I almost think the first link might be renamed to be more clear. It's not clear from that menu item what it does compared to the (www) link, perhaps. Maybe it could be re-labeled something like "Submit to HTML5 validator" or something along those lines instead...

Reply:
Or "Upload to HTML validator" to be really clear?

Reply:
Maybe "Validate current HTML (online)".

I guess I can sort of see a reason to have both, but I definitely was confused. I assume the CSS validation doesn't have an API component, since there's not a corresponding menu item for validating the current CSS?

Reply:
The W3C CSS service does have an API available, although we haven't explored it yet. It's a different style (SOAP/XML) than the Nu validator's API (REST/JSON). It wasn't discussed in #451 but I recall it came up in Slack at some point.

I'm not averse to looking at it, but I wouldn't say it's a priority either.

Reply:
Added #789 to look into that at some point.

Reply:
Maybe we don't need these two "www" buttons at all, especially if the SOAP/XML method works OK?


I originally said we didn't need the CSS validator, since the Nu HTML validator also reports errors in the CSS. But @tangledhelix thought it could be useful in order to be able to control whether validating to CSS3 or CSS2.1. So then I thought I'd add the link to the www version, and while I was doing it thought we might as well have a link to the HTML validator too. Then it was confusing to have two links to the validator, so maybe we don't need the www versions.
("And that's everything that happened in my life right up to the time I got this feedback" - Grandpa Simpson)

Reply:
If one can validate "in-app" then IMO there's no reason to have the links.

Actually, maybe the better plan is (now that we seem to be moving to have both submit-via-API tools), drop the links in this PR and instead just have a menu item to go to the PPWB. The PPWB main page links to the HTML and CSS validators, plus PG's ebookmaker, in addition to offerings its own tools.